selfdrive/car/docs_definitions.py

```diff
@@ -129,9 +125,9 @@ def get_column(self, column: Column, star_icon: str, footnote_tag: str) -> str:
     elif column == Column.MODEL and len(self.years):
       item += f" {self.years}"
 
-    footnote = get_footnote(self.footnotes, column)
-    if footnote is not None:
-      item += footnote_tag.format(self.all_footnotes[footnote])
+    footnotes = get_footnotes(self.footnotes, column)
+    if len(footnotes):
```

Comment:
`len()` is not needed here.   (`if footnotes` is idiomadic Python for a list)

Reply:
We do this countless places around openpilot, so this is sticking with that standard. Plus it's better to be more explicit even if we don't have to be.